go/vm/lfvm/converter.go

```diff
@@ -555,8 +556,9 @@ func createOpToOpMap() []OpCode {
 			continue
 		}
 
+		toImplement := []evm.OpCode{evm.PREVRANDAO, evm.PUSH0, evm.TLOAD, evm.TSTORE, evm.MCOPY, evm.BLOBHASH, evm.BLOBBASEFEE} // TODO implement for new revision support
```

Comment:
`PREFRANDAO` should probably not be in this list since it is actually covered (just named differently)